guidelines/outline.md

```diff
@@ -0,0 +1,123 @@
+
+## Handling cultural references, idioms, and slang
+
+	How are cultural references, idioms, and slang handled in your locale?
+
+	Are there resources for cultural references, idioms, and slang in your language?
+
+## Units and Unit Conversion
+
+	Does your locale use the metric system or the imperial system?
+
+### Date Format
+
+	How are the date formats for weeks and months expressed? Erase the comment markers around the example template or show the format with your own example.
+
+
+    | Name | Format | Example |
+    | ---- | ------ | ------- |
+    | Short | mm/dd/yy | 12/31/99 |
+    | Abbreviated | mmm dd | Dec 31 |
+    | Long | Month Date, Year | December 31, 1999 |
+
+### Calendar view
+
+	Which date is considered the first day of the week, Sunday or Monday?
+
+	Is Lunar calendar observed?  Other regional calendar observed?
+
+### Time Format
+
+	How is time expressed in your language?
+
+### Numerals
+
+	How are numerals and percentages expressed in your language? Erase the comment markers around the example template or show the format with your own example.
+
+| Separator | Character Name | Symbol | Example |
+| --------- | -------------- | ------ | ------- |
+| Decimal | Period | `.` | 1.23 |
+| Thousands | Comma | `,` | 1,234 |
+
+### Currency
+
+	What currency and symbols are used in your country/language for paid apps?
+
+### Units
+
+	Do you use the imperial, metric or nautical system for measuring weight, distance, etc.?
+
+### Address and Postal Code Format
+
+	What is the address format in your language? Erase the comment markers around one of the examples or show the format with your own example.
+
+	Example: Most Asian countries start from big to small:
+
+    [Country]
+    [postal code][state/province][city][district]
+    [street number and name][building and suite numbers]
+    [addressee]
+
+	Countries of European languages start from small to big:
+
+    [addressee]
+    [street number and name][building and suite numbers]
+    [district][city][state/province][postal code]
+    [Country]
+
+### Telephone Number format
+
+	How are telephone numbers expressed in your language? Erase the comment markers around the example template or show the format with your own example.
+
+`(###) ###-####`
+
+## Grammar
+
+### Spelling And Grammar Checks
+
+	Does your language have standard spelling or grammar checks?
+
+### Punctuation
+
+	Do you use different punctuation rules in your Mozilla localization than what your language standard defines?
+
+### Whitespace
+
+	Does your language require the use of white space around words, sentences, paragraphs, etc.?
+
+## Copy rules
+
+	Please leave the following line in your style guide unless it does not apply to your locale. You may translate it if your style guide is in your language
+
+Read about copy rules styles in the [General Mozilla L10n Style Guide](mozilla_general/README.md#copy-rules).
+
+	Is there an international/national standard for capitalization in your language?
+
```

Comment:
This section talks a lot about capitalization, we should specify that this is an area where people can list their regional specific styles, such as equivalent to English content, or other styles unique to their language. A lot of languages don't have capitalization at all:

    Bold letters
    All caps
    Book titles or article/blog title
    Italic
    Quotation mark

Reply:
Added something